api/v1/serializers/developers-serializer.js

```diff
@@ -0,0 +1,85 @@
+const appRoot = require('app-root-path');
+const decamelize = require('decamelize');
+const JsonApiSerializer = require('jsonapi-serializer').Serializer;
+const _ = require('lodash');
+
+const { serializerOptions } = appRoot.require('utils/jsonapi');
+const { openapi } = appRoot.require('utils/load-openapi');
+const { paginate } = appRoot.require('utils/paginator');
+const { apiBaseUrl, resourcePathLink, paramsLink } = appRoot.require('utils/uri-builder');
+
+const developerResourceProp = openapi.definitions.DeveloperResource.properties;
+const developerResourceType = developerResourceProp.type.enum[0];
+const developerResourceKeys = _.keys(developerResourceProp.attributes.properties);
+const developerResourcePath = 'developers';
+const developerResourceUrl = resourcePathLink(apiBaseUrl, developerResourcePath);
+
+/**
+ * The column name getting from database is usually UPPER_CASE.
+ * This block of code is to make the camelCase keys defined in openapi.yaml be
+ * UPPER_CASE so that the serializer can correctly match the corresponding columns
+ * from the raw data rows.
+ */
+_.forEach(developerResourceKeys, (key, index) => {
+  developerResourceKeys[index] = decamelize(key).toUpperCase();
+});
```

Comment:
If you use the alias to make your retrieved raw data be camelCase, then you don't need to decamelize it.

Reply:
added aliases and removed decamelize.